src/Umbraco.Core/PropertyEditors/IntegerPropertyEditor.cs

```diff
@@ -50,5 +52,65 @@ public IntegerPropertyValueEditor(
                 : int.TryParse(value?.ToString(), CultureInfo.InvariantCulture, out var parsedIntegerValue)
                     ? parsedIntegerValue
                     : null;
+
+        internal abstract class IntegerPropertyConfigurationValidatorBase : ConfigurationValidatorBase
+        {
+            protected static bool TryParsePropertyValue(object? value, out int parsedIntegerValue)
+                => int.TryParse(value?.ToString(), CultureInfo.InvariantCulture, out parsedIntegerValue);
+        }
+
+        internal class MinMaxValidator : IntegerPropertyConfigurationValidatorBase, IValueValidator
+        {
+            public IEnumerable<ValidationResult> Validate(object? value, string? valueType, object? dataTypeConfiguration, PropertyValidationContext validationContext)
+            {
+                if (TryParsePropertyValue(value, out var parsedIntegerValue) is false)
+                {
+                    yield break;
+                }
+
+                if (TryGetConfiguredValue(dataTypeConfiguration, "min", out int min) && parsedIntegerValue < min)
+                {
+                    yield return new ValidationResult($"The value {value} is less than the allowed minimum value of {min}", ["value"]);
```

Comment:
Quite clean, I like it, but I think this should be localized, at least it is for media :smile:

https://github.com/umbraco/Umbraco-CMS/pull/18429/files

Reply:
Agreed, I'll look at that.